src/spawn.coffee

```diff
@@ -35,9 +9,10 @@ interpolatePort = (port) ->
 
 module.exports = (name, command, commandArgs, port, logPath, logHandle, spawnOpts) ->
   commandArgs = commandArgs.map(interpolatePort port)
+  nodeArgs = [ WRAPPER ].concat [ command ], commandArgs
```

Comment:
Technically not but I think it adds clarity - e.g. it tells me that the argument `command` should never be an array. Otherwise someone using this library could start passing command and arguments into `command` because it works and happens to be easier. And then later on complain that we broke their code when that stops working.